### PR TITLE
OSDOCS-3342: Network Requirements

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR adds some network requirements to the cluster-wide proxy documentation. It also restructures the prereqs to include all general and network requirements into one module.

JIRA: https://issues.redhat.com/browse/OSDOCS-3342

PREVIEWS:
* **OSD**  https://deploy-preview-43022--osdocs.netlify.app/openshift-dedicated/latest/networking/configuring-cluster-wide-proxy.html#cluster-wide-proxy-prereqs_cluster-wide-proxy-configuration
* **ROSA**  https://deploy-preview-43022--osdocs.netlify.app/openshift-rosa/latest/networking/configuring-cluster-wide-proxy.html#cluster-wide-proxy-prereqs_cluster-wide-proxy-configuration

### CODE REVIEW

**Comment by mrbarge** [comment]
lgtm

**Comment by mrbarge** [comment]
@EricPonvelle 
Sorry to be a pain, but I'd like one more table row entry added for the "SRE and management URLs" table.
I have added it to the OSDOCS-3342 ticket description.

File: modules/cluster-wide-proxy-preqs.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+|`observatorium-mst.api.openshift.com`
+|https/443
+|Required. Used for Managed OpenShift-specific telemetry.
+
+|`sso.redhat.com`
+|https/443
+|The https://cloud.redhat.com/openshift site uses authentication from sso.redhat.com to download the cluster pull secret and use Red Hat SaaS solutions to facilitate monitoring of your subscriptions, cluster inventory, and chargeback reporting.
+|===
++
+** Allowlist the following site reliability engineering (SRE) and management URL for re-encryption.
```
Comment (mburke5678) [inline modules/cluster-wide-proxy-preqs.adoc:43]: s/URL/URLs ??

File: modules/cluster-wide-proxy-preqs.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+`inputs8.osdsecuritylogs.splunkcloud.com`
+`inputs9.osdsecuritylogs.splunkcloud.com`
+`inputs10.osdsecuritylogs.splunkcloud.com`
+`inputs11.osdsecuritylogs.splunkcloud.com`
+`inputs12.osdsecuritylogs.splunkcloud.com`
+`inputs13.osdsecuritylogs.splunkcloud.com`
+`inputs14.osdsecuritylogs.splunkcloud.com`
+`inputs15.osdsecuritylogs.splunkcloud.com`
+|tcp/9997
+|Used by the splunk-forwarder-operator as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
```
Comment (mburke5678) [inline modules/cluster-wide-proxy-preqs.adoc:67]: s/logging forwarding endpoint/log forwarding endpoint ? When I was on the logging team, the term was log forwarding. Maybe it changed?

File: modules/cluster-wide-proxy-preqs.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+|`http-inputs-osdsecuritylogs.splunkcloud.com`
+|https/443
+|Used by the splunk-forwarder-operator as a logging forwarding endpoint to be used by Red Hat SRE for log-based alerting.
```
Comment (mburke5678) [inline modules/cluster-wide-proxy-preqs.adoc:71]: Change here if you change [there](https://github.com/openshift/openshift-docs/pull/43022/files#diff-efc126ca8b4df292e019ccdca549e4bf3321b2175075aa8fdc4278f8effa8623R67).

**Comment by mburke5678** [comment]
@EricPonvelle A few nits. Otherwise LGTM

**Pull request merged**